settings/resource_notification_destination.go

```diff
@@ -0,0 +1,177 @@
+package settings
+
+import (
+	"context"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/settings"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func setStruct(s *settings.NotificationDestination, readND *settings.NotificationDestination) {
+	switch readND.DestinationType {
+	case settings.DestinationTypeSlack:
+		readND.Config.Slack.Url = s.Config.Slack.Url
```

Comment:
we need to add a check that `s.Config.Slack` (and other blocks) aren't null - when we importing the resource, source data doesn't exist, so it will crash with null pointer